package.json

```diff
@@ -13,15 +13,18 @@
     "email": "contact@foxglove.dev",
     "url": "https://foxglove.dev/"
   },
-  "main": "dist/index.js",
+  "module": "dist/esm/index.js",
+  "main": "dist/cjs/index.js",
   "files": [
     "dist",
     "src"
   ],
   "scripts": {
-    "build": "tsc -b",
-    "prepack": "tsc -b",
-    "clean": "tsc -b --clean",
+    "build": "npm run build:cjs && npm run build:esm",
+    "build:esm": "tsc -b tsconfig.json",
+    "build:cjs": "tsc -b tsconfig.cjs.json",
+    "prepack": "npm run build",
+    "clean": "tsc -b tsconfig.cjs.json --clean && tsc -b tsconfig.json --clean",
```

Comment:
this would be simpler as `rimraf dist *.tsbuildinfo`

Reply:
Clean also removes the built .js and .d.ts files tho, right?

Reply:
Hm, so I guess this only cleans files that the current tsconfig would build, right? Like if you rename a .ts file then will clean delete the .js with the old name?

Reply:
All the built files are going into `dist` so in this case it's the same thing. If you are building `.js` files into your `src` directory then `tsc -b --clean` is necessary.

Reply:
https://github.com/foxglove/template-typescript/pull/21

Its quite unfortunate that tsc puts the buildinfo into the dest dir - tho equally unfortunate when it puts it in to the src dir too.